presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -1044,4 +1046,16 @@ public boolean isPushdownSubfieldsEnabled()
     {
         return pushdownSubfieldsEnabled;
     }
+
+    public boolean isTableWriterMergeOperatorEnabled()
+    {
+        return tableWriterMergeOperatorEnabled;
+    }
+
+    @Config("experimental.table-writer-merge-operator-enabled")
+    public FeaturesConfig setTableWriterMergeOperatorEnabled(boolean tableWriterMergeOperatorEnabled)
```

Comment:
nit: do you want to have `ConfigDescription` ? :)

Reply:
This flag is more of a safety flag for rollout. It is not intended to be switched back and forth. At some point it has to be removed completely.